docs/process/interruptive-surveys.md

```diff
@@ -1,26 +1,80 @@
+
+### SurveyGizmo Access
+
+We have heavily restricted access to creating and editing surveys for FxA. In order to edit surveys in FxA, you will need to do the following:
+
+1. Sign in to SurveyGizmo using [sso.mozilla.com](https://sso.mozilla.com)
+2. Set up 2FA inside SurveyGizmo
+3. Be added to the `FxA_Product` group by one of the FxA PMs, or a SurveyGizmo admin
+
+In general though, PMs and UR folks can set up surveys and provide appropriate URLs to launch surveys.
+
+### For survey builders
+
+In order to build appropriately formatted surveys, you will first need access to the `FxA_Product` group in SurveyGizmo.
+
+#### Creating a new survey
+
+* Look in the SurveyGizmo sidebar for `FxA Interrupt Surveys`
+* In this folder, you will see a survey called `FxA Interrupt Template`, by clicking on the vertical ellipse at the end of the row, you will be able to make a copy of this survey.
+* When you create the survey, please make sure you set the team to `FxA_Product`
+
+#### Required Survey Bits
+
+If you're using the standard template, you will see two special "Actions" in the survey builder.
+
+* **Hidden Value Action** - This is the first field on the survey. Hidden from users it is intended to capture query param data appended to each survey by FxA. This field is not currently used, but the FxA team will be adding parametric data to surveys soon.
+* **Javascript Action** - **IMPORTANT: If you delete this action your survey will not close correctly.** This is located on the last page of the survey after the last question. This field contains a bit of custom JavaScript that creates a postMessage event that FxA listens for to handle survey completion correctly. If you change or edit your survey to remove pages, please make sure this JS Action stays on the last page of your survey with questions present!
+
+#### Formatting & Styling Surveys
+
+As of this writing, FxA interrupt surveys are only formatted to accept radio, checkbox, and open textarea style questions in SurveyGizmo. Please limit one question per page.
+
+Other question types may present formatting issues due to the small size of the survey as presented to users.
+
+If you need to update the CSS of your survey to accommodate a new question type, please make sure to update the CSS file in both your survey as well as the original `FxA Interrupt Template` CSS file. In this way, your improvements will carry forward to future surveys.
+
+You can can edit survey CSS from the editor by going to the `Style` tab. From here, look for the `HTML/CSS editor` in the lower right. Please do not mess with the GUI theme editor options on the left side of the screen.
 
 ## Configuration Options
+
 * `id` - string - description of the survey, using dashes instead of spaces, to make the survey easier to grok for other teammates.
 * `view` - string - view name, where the survey should show up to user
 * `rate` - integer - NOT IMPLEMENTED rate at which the survey should be shown to qualifying users
```

Comment:
I think you can take out "NOT IMPLEMENTED" now.

Also, is it possible to add instructions to pull up the survey for local development around the environment variable (if still applicable, I'm not super involved here), these showing up in localStorage, and changing the rate to like `1` to make sure it shows up every time?

Reply:
There's an issue for that! https://jira.mozilla.com/browse/FXA-2024